config/routes.rb

```diff
@@ -109,4 +109,6 @@
   # High voltage configuration for static pages. Matches routes from the root of the domain. Uses
   # HighVoltage::Constraints::RootRoute to validate that the view exists.
   get "/*id" => "pages#show", :as => :page, :format => false, :constraints => HighVoltage::Constraints::RootRoute.new
+
+  match "*unmatched", to: "application#not_found_error", via: :all
```

Comment:
I've never seen `match` or `*unmatched` before?! How did you find those?

Reply:
Second that!

Reply:
I've updated the commit message to add this info.